### PR TITLE
fix: #5

### LINKED ISSUE #5: pullreqdemo.py is screwed up
@tkrabec screwed up pullreqdemo.py... bloody typical!

### PR DESCRIPTION
**Pull request by databeast**
fixed up pullreq.py
added default file

fix: #5

### CODE REVIEW

**Comment by databeast** [comment]
this doesn't merge.

please go back and fix it, ya dummy.

**Comment by tkrabec** [comment]
So I'm not the only dummy here :) ha

**Comment by databeast** [comment]
- this is a 
- bullet
- list

File: pullreqdemo.py [hunk 1]
```diff
@@ -11,6 +11,7 @@ def filetest(name='default.txt'):
     f.close()
     
 if __name__ == '__main__':
-    filetest(sys.argv[1]) # found missing )
+    filetest(sys.argv[1])
+    print "ok look this changed!"
```
Comment (databeast) [inline pullreqdemo.py:15]: hey are you sure this is a good idea?

**Pull request closed without merging**